### ISSUE TITLE
webin-cli version 8.0.0 incorrectly returns with `Invalid submission account user name or password`

### ISSUE THREAD

**Issue by anna-parker** [enasequence/webin-cli#127]
I have been using the webin-cli 7.3.1 for the last couple months, I now changed to the new 8.0.0 version and I am unable to submit sequences, 
```
java -jar webin-cli.jar -username $ena_submission_username -password $ena_submission_password -context genome -manifest manifest_file -submit -test
```
Fails with the error 
```
ERROR: FTP service authentication error. Invalid submission account user name or password. Please try enclosing your password in single quotes. Failed to upload files to server because of a user error.
```
The command works on the 7.3.1 version and my password and username are correct. My password is enclosed in single quotes.

**Comment by corneliusroemer** [comment]
I tried to reproduce but I can't get the "FTP service authentication error" part of the message to show. If the password is just wrong, I get:

```log
ERROR: Invalid submission account user name or password. Please try enclosing your password in single quotes.
```

It's possible that this was a temporary server side issue, potentially related to the test instance as well. Here are the files I tried this with:
[assembly.zip](https://github.com/user-attachments/files/17650909/assembly.zip)

Let's hope this has been resolved.
